02_Instructor_Setup.Rmd

```diff
@@ -0,0 +1,268 @@
+# (PART\*) Instructor Setup {-}
+
+```{r, include = FALSE}
+ottrpal::set_knitr_image_path()
+```
+
+# Overview 
+
+<br>
+
+The setup instructions are presented in two parts:
+
+- **Instructor Setup:** covers everything needed to set up yourself and any co-instructors or TAs to work on AnVIL.  Depending on your team and funding, *you may only need to do these steps once* (and then make updates on an as-needed basis for changes in team members or funding).
+- **Running the Class:** covers how to grant your students access to AnVIL.  These steps will need to be *repeated for each new offering of the course*.
+
+## What do I need to do?
+
+You may not need to do everything in the instructor setup section!
+
+This section contains instructions for setting up billing from scratch.  If your funding is being managed by a third party (e.g. through a funding mechanism such as [STRIDES](https://datascience.nih.gov/strides) or through your institution) you can skip some steps.  The table below provides guidance on which steps you will need to complete:
+
+| Task | Self-managed Funding | 3rd-party Funding |
```

Comment:
Love this - I like the idea of creating a table to help folks navigate our more involved or longer content